conf/db/migration/default/V6__Add_threadpool_queue.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE thread_pool_config
+ADD COLUMN queue_size INTEGER
+  NOT NULL
+  DEFAULT 256
```

Comment:
If default in the DB is 256, does the default of 0 in code make sense when this is actively checking for queue being greater than 0?